### ISSUE TITLE
Queue full 

### ISSUE THREAD

**Issue by Kennynsandy** [sologenic/sologenic-expo-wallet#156]
What’s going on with Sologenic. Can’t buy solo with XRP. “Not Validating” & “Queue full” (ALL day). Who can fix this? Probably junk coins being dropped on the ledger. How can this be remedied without hindering true creators?

**Comment by michaelmcfann26** [comment]
Yes some help would be nice I’ve tried everything to go ahead a get my funds of this DEX.

**Comment by michaelmcfann26** [comment]
Start blasting on Twitter @realsologenic we all need to come togwther and flood Twitter
